src/Features/CSharp/Portable/ExtractMethod/Extensions.cs

```diff
@@ -261,6 +262,12 @@ public static bool IsExpression([NotNullWhen(returnValue: true)] this SyntaxNode
             return node is ExpressionSyntax;
         }
 
+        public static bool IsExpression([NotNullWhen(returnValue: true)] this SyntaxNode? node, [NotNullWhen(true)] out ExpressionSyntax? expression)
```

Comment:
>IsExpression [](start = 27, length = 12)

Is there only one caller? If so, perhaps inline this method since it's so simple.

Reply:
Inlined the original method, simplified the resulting code